src/desktop/apps/user/templates/layout.jade

```diff
@@ -12,11 +12,8 @@ block body
     class='js-settings-page'
   )
     .main-layout-container
-      header.settings-page__header
-        h1.settings-page__header__name
-          = user.get('name')
-
-        include ../components/tabs/index
```

Comment:
This template file can probably be removed?

Reply:
👍  This is a bit hard for me to follow/test. I will remove it in a follow-up ticket